src/modules/categories/components/category/category.jsx

```diff
@@ -41,7 +41,7 @@ export default class Category extends Component {
     const s = this.state
 
     const isNullCategory = p.category === 'null-category' && p.popularity === 0
-    const roundedPop = new BigNumber(p.popularity).integerValue(BigNumber.ROUND_HALF_EVEN)
+    const roundedPop = new BigNumber(p.popularity.toString()).integerValue(BigNumber.ROUND_HALF_EVEN)
     let popString = roundedPop.toNumber() === 1 ? ' SHARE' : ' SHARES'
```

Comment:
Don't use JS numbers, suggest instead:
```javascript
let popString = roundedPop.toFixed() === '1' ? ' SHARE' : ' SHARES'
```

Reply:
existing code, will rebase to see if fixed as part of bignumber changes. if not will fix.